Tutorials/whatiswot/8-JSON_Schema_in_Practice_3/8-JSON-Schema-3.txt

```diff
@@ -0,0 +1,44 @@
+In the previous videos of JSON Schema, we have shown the features and basic implementations of JSON Schema. In this video, we will deliver the actual validation process with libraries and examples.
+
+Throughout this video, we will mainly focus on validation with the JavaScript library AJV and the Python library          jsonschema, but there are many other libraries in different programming languages and tools to validate JSON Schema. 
+
+Here are some of the popular ones in different programming languages.  There are many more validators. For further information, you can check the JSON schema website.
+
+This  is the basic validation flow that all libraries follow.
+The library takes a schema for your JSON data and validates your data according to the schema. Outputs the validation result as a boolean 
+
+So let's start with AJV! It is a JavaScript library.
+
+This is the same temperature sensor example we used before. Let’s examine this in JavaScript code.
+
+First, We are importing the AJV. 
+
+This is the Schema that describes the JSON. 
+
+With this line of code, AJV takes schema as a parameter compiles it to function, and caches it.
+
+This is the data to be validated. 
+
+This step actually validates the data and writes the result into a valid boolean. 
+
+This step is for printing the result of the validation. If valid is true, then it prints "JSON Schema is valid". Otherwise, it prints the "JSON Schema is not valid." and the error message. 
+
+It outputs that schema is valid since the data that we enter suits the predefined schema. 
+
+Now let's take a look at the same JSON Schema example in Python jsonschema. It is an implementation of the JSON Schema specification for Python. 
+
+We will use the same temperature example that we use for AJV but for Python this time.
+Let's take a look at the code in detail. 
+
+With this line of code, we are importing the function from the library. 
+
+This is the same schema in the previous example. 
+
+This is the data to be validated. 
+
+This validate function takes the data and the schema validates it and prints the result. If the data is valid, it does not print anything. Otherwise, it prints the error message. 
+
+The code outputs that the data is not valid since the property "temperatureOf" needs to be string. However, in the given data it is an integer. 
+
+In the next video, we will talk about JSON and what it is detailly.
```

Comment:
Next video is definitely not JSON. We have OSI Layers in the list but maybe the semantic web part will come there. To avoid changing the video, we can simply say something like below:

```suggestion
This concludes the videos on JSON Schema. You can continue with the tutorial to learn more about the Web of Things.
```